### PR TITLE
add READMEs to dapr/proto folders for ease of information

### PR DESCRIPTION
**Pull request by cicoyle**
# Description

I added in READMEs to the `dapr/proto/*` folders to make it clearer what the APIs in the folders were for. I know there is a README in `dapr/proto`. I felt as though we may benefit from folder specific READMEs, as it was unclear that the protos in the `dapr/proto/components` were specifically for pluggable components with things as is. I then didn't want to just have a README in the `dapr/proto/components`, so I added one for each of the folders with a relevant service doc link. Feel free to comment if you feel the wording could be improved or expanded upon.

### CODE REVIEW

File: dapr/proto/sentry/v1/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Sentry Service APIs
+
+This folder is intended for Sentry Service APIs that the Daprd Sidecars use to manage mTLS between services and acts as a certificate authority. [See more on the Sentry Control Plane Service in the Dapr Docs](https://docs.dapr.io/concepts/dapr-services/sentry/).
```
Comment (ItalyPaleAle) [inline dapr/proto/sentry/v1/README.md:3]: Sentry is used by the Dapr control plane services too

File: dapr/proto/runtime/v1/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Public Dapr Runtime APIs
+
+This folder is intended for User facing APIs.
```
Comment (ItalyPaleAle) [inline dapr/proto/runtime/v1/README.md:3]:
```suggestion
This folder is intended for user-facing APIs. 
```

File: dapr/proto/runtime/v1/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Public Dapr Runtime APIs
+
+This folder is intended for User facing APIs. 
+
+The protos in `dapr.proto` are intended for users to access the Dapr Building Blocks. The `appcallback.proto` is intended for users that need to interact with the Dapr runtime.
```
Comment (ItalyPaleAle) [inline dapr/proto/runtime/v1/README.md:5]:
```suggestion
- `dapr.proto` is used by for the services implemented by the Dapr runtime. Apps calling into the Dapr runtime use these services.
- `appcallback.proto` is for services implemented by apps to receive messages from the Dapr runtime.
```

**Review by ItalyPaleAle — CHANGES_REQUESTED** [review]
Since you're doing this, can I also recommend adding some links to:

1. How to re-generate the code after making changes to the protos (`make gen-proto`)
2. Where in the source code the generated files are: `pkg/proto`

File: dapr/proto/sentry/v1/README.md (hunk 1 above)
Comment (daixiang0) [reply dapr/proto/sentry/v1/README.md:3]:
```suggestion
This folder is intended for `sentry` service APIs that the `daprd` sidecars use to manage mTLS between services and acts as a certificate authority. More details about the `sentry` please refer to the [docs](https://docs.dapr.io/concepts/dapr-services/sentry/).
```

**Comment by cicoyle** [comment]
> Since you're doing this, can I also recommend adding some links to:
> 
> 1. How to re-generate the code after making changes to the protos (`make gen-proto`)
> 2. Where in the source code the generated files are: `pkg/proto`

I added this per README, by basically copying/tweaking the instructions from the `/dapr` folder

**Review by ItalyPaleAle — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**